backend/main.py

```diff
@@ -48,9 +48,12 @@ def get_cards(
     return crud.get_cards(db)
 
 
-@app.post("/cards", status_code=201, response_model=Card)
-def create_card(card: CardCreate, db: Session = Depends(get_db)) -> models.Room:
-    return crud.create_card(db, card)
+@app.get("/cards/{name}", response_model=Card)
+def get_card_by_name(
+    name: str,
+    db: Session = Depends(get_db),
+) -> List[models.Card]:
```

Comment:
Why is this a List? Shouldn't this be just one card?

Reply:
:shrug: copy pasta from a few lines down. I'll send a patch

Reply:
pushed